barretenberg-wasm.nix

```
@@ -1,11 +1,13 @@
-{ stdenv, cmake, ninja, binaryen, callPackage }:
+{ lib, stdenv, cmake, ninja, binaryen, callPackage, debug ? false }:
```

```suggestion
{ lib, stdenv, cmake, ninja, binaryen, callPackage, DEBUG ? false }:
```
Thoughts on making this all caps to distinguish from packages we're getting from nixpkgs?

---

not a regular practice I think but no objections